tools/pepsi/src/pre_game.rs

```diff
@@ -65,7 +67,7 @@ pub async fn pre_game(arguments: Arguments, repository: &Repository) -> Result<(
 
     recording(
         RecordingArguments {
-            cycler_instances_to_be_recorded: arguments.cycler_instances_to_be_recorded,
+            recording_settings: arguments.recording_settings,
```

Comment:
Naming: Recording frequencies? Then you can also shorten the comment of the command line argument.

Reply:
Frequency is something divided by time. I guess this would be a period or interval

Reply:
I renamed this to `recording_interval` now.